circle.yml

```diff
@@ -1,36 +1,210 @@
-machine:
-  services:
-    - docker
-
-dependencies:
-  override:
-    - docker build -t cyclus/cycamore:latest .
-
-test:
-  override:
-    - docker run cyclus/cycamore:latest cycamore_unit_tests
-    - docker run cyclus/cycamore:latest nosetests -w /cycamore/tests
-
-deployment:
-  develop:
-    branch: develop
-    commands:
-      - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
-      - docker push cyclus/cycamore:latest
-
-  master:
-    branch: master
-    commands:
-      - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
-      # Update cyclus/cylus:stable docker container
-      - rm Dockerfile && ln -s docker/master-ci/Dockerfile # place the correct Dockerfile
-      - docker build -t cyclus/cycamore:stable .
-      - docker push cyclus/cycamore:stable
-      # DEB GENERATION
-        #ubuntu 14.04
-      - docker/deb-ci/build_upload_deb.sh 14 :
-          timeout: 1200 # set 20 minute timeout (to avoid CI to fail)
-          # Ubuntu 16.04
-      - docker/deb-ci/build_upload_deb.sh 16 :
-          timeout: 1200 # set 20 minute timeout (to avoid CI to fail)
-        ## END DEB GENERATION
+version: 2
+
+jobs:
+    # Build Cyclus
+    build:
+        docker:
+            - image: cyclus/cyclus:latest
+        working_directory: ~/cycamore
+        steps:
+            - checkout
+            - run:
+                name: Build Docker Image
+                command: |
+                    python install.py -j 2 --build-type=Release \
+                    -DBLAS_LIBRARIES="/opt/conda/lib/libblas.so" \
+                    -DLAPACK_LIBRARIES="/opt/conda/lib/liblapack.so"
```

Comment:
Isn't it cleaner to just have this in the Dockerfile rather than in both places?

Reply:
I did not find a clean way yet. 
the idea of CircleCI 2.0 can run docker... so most of the Dockerfile, we have, are not required anymore. (I need to figure out which one and do some cleaning) -- The only Dockerfile needed is for the update on DoockerHub 

relying on CircleCI direct Docker is apparently faster (according to what they say) and allow us to separate the different jobs: building - testing - deploying without having to push the container all the time.

also the naming part is not correct... it should be Build Cyclus

Reply:
Yeah, Sorry, I was misunderstanding the intent here.